lib/support/nginx/gitlab

```diff
@@ -114,6 +114,9 @@ server {
   }
 
   location ~ ^/[\w\.-]+/[\w\.-]+/gitlab-lfs/objects {
+    ## If you use relative url, please uncomment below line and remove origin line.
+    # location ~ ^/YOUR_RELATIVE_URL/[\w\.-]+/[\w\.-]+/gitlab-lfs/objects {
```

Comment:
We should not repeat the entire regex in the comment, that is a maintenance hassle. We should rather say something like "change `^/` to `^/gitlab/`.